### PR TITLE
Added linear schedule for learning rate

### PR DESCRIPTION
**Pull request by Piggelinus**
This attempts to solve #31 
I've added a simple implementation of a linear schedule on the learning rate from an initial value down to 0.
This is mentioned in the ACKTR paper but was missing here.
There is now an argument for toggling it (off by default)

There will be more elegant solutions based on http://pytorch.org/docs/master/_modules/torch/optim/lr_scheduler.html#LambdaLR, but I had trouble implementing this because of an older version of pytorch (0.2).

### CODE REVIEW

**Comment by ikostrikov** [comment]
Hi! Sorry for the delay, I've just got back from NIPS. I will review it this week.
